### DISCUSSION TITLE
В истории виджета home assistant показывается не температура, а погода (например, солнечно, облачно), можно это поправить?

### DISCUSSION THREAD

**Discussion by Nerd0o0** [IATkachenko/HA-YandexWeather#17] [Q&A] [answered]

**Comment by IATkachenko** [comment, marked as answer]
Нет, так работает Home Assistant: состояние погоды это солнечно, облачно итд, а температура, влажность, давление итд -- атрибуты. На сколько я знаю, история атрибутов не записывается.

Если нужна история температуры (давления, влажности, скорости и направления ветра) -- активируйте соответствующий сенсор (он деактивирован по умолчанию) на странице устройства интеграции.

**Comment by Nerd0o0** [comment]
Активированы, добавлены в виджет на главной странице, кликая в нужное место можно увидеть историю температуры. Спасибо, это именно то, что мне нужно
